content/rs/technology-behind-redis-enterprise.md

```diff
@@ -1,102 +1,73 @@
-(out/in) without losing data.
+
+Redis Enterprise was built to simplify operations while providing seamless scalability, top performance, and high availability.
+
+Here are some of the features and design decisions that make this possible.
+
+## Data access and cluster management
+
+A Redis Enterprise Cluster separates data access from cluster
+management. This decoupled architecture results in increased reliability,
+top performance, and simplified operations.
+
+### Data access
+
+A Redis Enterprise Cluster hosts one or more Redis databases. Access to these databases runs through a multi-threaded proxy that lives on each cluster node. These proxies efficiently and transparently route queries to the appropriate underlying Redis instance.
+
+You can use any standard Redis client to access a Redis Enterprise database via the proxy. The proxies also support cluster-aware clients for applications requiring the lowest possible latency.
+
+In addition, the proxies understand Memcached's text and binary protocols. Applications using Memcached can immediately start using Redis Enterprise and take advantage of its many features, including built-in replication, auto-failover, data persistence, backups, and linear scaling.
 
 ### Cluster management
 
-The cluster manager is a sophisticated governing function that provides
-capabilities such as resharding, rebalancing, auto-failover,
+This cluster manager consists of a number of software components that monitor
+and configure the many Redis instances that make up a cluster.
+
+The cluster manager handles resharding, rebalancing, auto-failover,
 rack-awareness, database provisioning, resource management, data
-persistence configuration, backup and recovery. The cluster manager
-employs multiple watchdog mechanisms at the cluster node level and at
-the process level, which guarantees an instant response to events such
-as node, rack and data center failures, and can handle multiple failover
-events at the same time. Because the cluster manager is entirely
-decoupled from the data path components, changes to its software
-components do not affect the data path components.
-
-## Consistent top performance
-
-Multiple techniques are implemented by RS to provide consistent top
-performance for Redis:
-
-- A shared-nothing architecture maximizes the performance of each
-    database. Automatic migration of shards between nodes is performed
-    when needed.
-- The zero-latency proxy utilizes several mechanisms for improving
-    performance, including just-in-time pipelining, socket connections,
-    connection pooling, and multiplexing.
-- Improvements to AOF data persistence and optimized rewrite
+persistence configuration, backup, and recovery.
+
+## High performance and availability
+
+Redis Enterprise uses several techniques to optimize performance and availability.
+
+- The cluster is a shared-nothing architecture, which increases reliability and availability, and makes it easy to add and remove nodes.
```

Comment:
Maybe: The cluster uses a shared-nothing architecture that increases reliability and availability. This architecture also makes it easy to add and remove nodes.

Reply:
I took "uses", which is definitely better.